examples/pom.xml

```diff
@@ -221,14 +221,21 @@
       <groupId>com.github.cukespace</groupId>
       <artifactId>cukespace-core</artifactId>
       <version>${project.version}</version>
-      <scope>test</scope>
       <exclusions>
         <exclusion>
           <groupId>com.google.guava</groupId>
           <artifactId>guava</artifactId>
         </exclusion>
       </exclusions>
+      <scope>test</scope>
     </dependency>
+    
+    <dependency>
```

Comment:
I'm actually running it from JBoss Developer Studio 7.1.1 in windows, so it's being run from the maven plugin that comes with it